### PR TITLE
"updated leaderboard css and page functionality"

### PR DESCRIPTION
**Pull request by Sibusiso-Retro**

### CODE REVIEW

File: skietbaan_fe/src/components/LeaderboardPage.js [hunk 1]
```diff
@@ -434,7 +442,7 @@ class LeaderboardPage extends Component {
             /* Leaderboard Page Main Container */
             <div className="leaderboard-container">
                 <div className="row justify-content-center">
-                    <div className="col-sm-8 text-left">
+                    <div className="col-sm-8 text-left" style={{backgroundColor:"#F5F5F5"}}>
                         <div className="closed-filter-section">
```
Comment (kyle-manganyi) [inline skietbaan_fe/src/components/LeaderboardPage.js]: do you need inline styling?

File: skietbaan_fe/src/components/LeaderboardPage.js [hunk 2]
```diff
@@ -81,6 +82,13 @@ class LeaderboardPage extends Component {
         }
         this.props.fetchleaderboadtabledata(filterSelection);
     }
+    getclickableSpaceSize(){
```
Comment (kyle-manganyi) [inline skietbaan_fe/src/components/LeaderboardPage.js]: not camelCase

File: skietbaan_fe/src/components/LeaderboardPage.js (hunk 1 above)
Comment (Sibusiso-Retro) [reply skietbaan_fe/src/components/LeaderboardPage.js]: yeah definetely

Comment (kyle-manganyi) [reply skietbaan_fe/src/components/LeaderboardPage.js]: okay change camelCase one

File: skietbaan_fe/src/components/LeaderboardPage.js (hunk 2 above)
Comment (Sibusiso-Retro) [reply skietbaan_fe/src/components/LeaderboardPage.js]: resolved

**Pull request merged**